pygaggle/settings.py

```diff
@@ -5,13 +5,28 @@
 
 
 class Settings(BaseSettings):
-    cord19_index_path: str = 'data/lucene-index-covid-paragraph'
-
     # T5 model settings
-    t5_model_dir: str = 'gs://neuralresearcher_data/covid/data/model_exp304'
     t5_model_type: str = 't5-base'
     t5_max_length: int = 512
 
     # Cache settings
     cache_dir: Path = Path(os.getenv('XDG_CACHE_HOME', str(Path.home() / '.cache'))) / 'covidex'
     flush_cache: bool = False
+
+
+class MsMarcoSettings(Settings):
+    msmarco_index_path: str = '/content/data/index-msmarco-passage-20191117-0ed488'
```

Comment:
Should we convert all these as arguments to our main functions? It is hard to keep track of which exact command we used to run an experiment when they are in the code

Reply:
Hmm, I just followed similar settings as that which was included for CovidQA. Maybe things like the index_path which rarely every change can remain here, and we can move model directory to main?

Reply:
SGTM!